src/attributes/attributes.js

```diff
@@ -0,0 +1,49 @@
+(function () {
+	htmx.defineExtension('attributes', {
+		handleSwap: function (swapStyle, target, fragment) {
+			var source = fragment.firstElementChild || fragment;
```

Comment:
I was trying to work out what this firstElementChild alternative source option here was trying to do and why you are checking for hx-swap-oob and doing a actualTarget lookup.  I don't think i needed this when I was testing oob attribute swaps with my earlier test version i did as the handleSwap is called once for main swaps and then again for each oob swap and so it shouldn't really need extra handling logic.  But maybe you found an edge case you needed to handle